pyproject.toml

```diff
@@ -40,13 +40,20 @@ types-requests = "^2.31.0.2"
 types-pillow = "^10.0.0.2"
 dash-dangerously-set-inner-html = "^0.0.2"
 litellm = "0.1.607"
-torch = ">=2.0.0, !=2.0.1"
-transformers = "^4.33.0"
+torch = ">=2.0.0"
```

Comment:
If this is optional, it should be placed to an extra dependency group. If users want to use it, it must be installed by a different way.

For example, a nomal install is:

```sh
pip install yival
```

Then the specific install should be:

```sh
pip install yival[ml]
```

The `ml` above is an extra dependency group, short for **Machine Learning** as an example.

Reply:
I put these extra dependency in tool.poetry.extras:
<img width="425" alt="image" src="https://github.com/YiVal/YiVal/assets/55043304/277d6d4d-8152-4cd3-adb2-37d58b1432d5">

This seems ok ?